File: Assets/FirefighterScripts/WebcamFeed.cs
using UnityEngine;
using System.Collections;

public class WebcamFeed : MonoBehaviour {

	WebCamDevice[] cameras;
	WebCamTexture camfeed;
	
	public void Start()
	{
	  cameras = WebCamTexture.devices;
	
	  if ((cameras != null) && (cameras.Length > 0))
	  {
	    camfeed = new WebCamTexture(cameras[0].name, 640, 480, 30);
	 
	    camfeed.Play();
	
	    GameObject bgimg = GameObject.Find("Background Image");
	    bgimg.guiTexture.texture = camfeed;
	  }
	}
}

File: Assets/FirefighterScripts/WebcamFeed.js
#pragma strict

var cameraOutput : GUITexture; 
 
private var webcamTexture : WebCamTexture; 
private var devices : WebCamDevice[];
 
function Start () { 
    devices = WebCamTexture.devices; 
    webcamTexture = new WebCamTexture(); 
    if(devices.length > 0){ 
       webcamTexture.deviceName = devices[0].name; 
       webcamTexture.requestedWidth = Screen.width; 
       webcamTexture.requestedHeight = Screen.height; 
       var width = webcamTexture.requestedWidth; 
       var height = webcamTexture.requestedHeight;
       var centerX = (width/2) *-1;
       var centerY = (height/2) * -1;
       cameraOutput.pixelInset = Rect(centerX, centerY, width, height);
       cameraOutput.texture = webcamTexture; 
       webcamTexture.Play(); 
    } 
}
function OnGUI () { 
    if (WebCamDevice == null){ 
       GUI.Label(Rect(10,10,65,25),"NO ACTIVE CAMERA"); 
    }
}